### PR TITLE
fix: update colors@1.4.0 cli-table2@0.6.1 

### LINKED ISSUE #1884: security vulnerability in colors@1.4.1 and colors@1.4.44-liberty
**Describe the bug**
A Security Vuln was identified in the Colors package for >1.4.0, offending packages being `1.4.1`, `1.4.44-liberty`
- [source1](https://twitter.com/snyksec/status/1480286811482206216?ref_src=twsrc%5Egoogle%7Ctwcamp%5Eserp%7Ctwgr%5Etweet)
- [source2](https://twitter.com/snyksec/status/1480286811482206216?ref_src=twsrc%5Egoogle%7Ctwcamp%5Eserp%7Ctwgr%5Etweet)
- [source3](https://security.snyk.io/vuln/SNYK-JS-COLORS-2331906)

### PR DESCRIPTION
**Pull request by mannyluvstacos**
A Security Vuln was identified in the Colors package for >1.4.0, offending packages being `1.4.1`, `1.4.44-liberty`
- [source1](https://twitter.com/snyksec/status/1480286811482206216?ref_src=twsrc%5Egoogle%7Ctwcamp%5Eserp%7Ctwgr%5Etweet)
- [source2](https://twitter.com/snyksec/status/1480286811482206216?ref_src=twsrc%5Egoogle%7Ctwcamp%5Eserp%7Ctwgr%5Etweet)
- [source3](https://security.snyk.io/vuln/SNYK-JS-COLORS-2331906)

closes #1884

This PR pins the color package to `1.4.0` as advised on the [snyk page](https://snyk.io/blog/open-source-maintainer-pulls-the-plug-on-npm-packages-colors-and-faker-now-what/)
# Checklist:

- [x] I have read the [**CONTRIBUTING**](../CONTRIBUTING.md) document.
- [ ] My code follows the code style of this project.
- [ ] I have added tests to cover my changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have added an entry to the "Unreleased" section of the [**CHANGELOG**](../CHANGELOG.md), linking to this pull request.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -194,7 +195,7 @@
     "assertion-error-formatter": "^3.0.0",
     "capital-case": "^1.0.4",
     "cli-table3": "^0.6.0",
```
Comment (jan-molak) [inline package.json:197]: Please upgrade cli-table3 to 0.6.1 too - see https://github.com/cli-table/cli-table3/pull/251

Comment (mannyluvstacos) [reply package.json]: Updated!

**Comment by jan-molak** [comment]
@davidjgoss - any chance this could be back-ported to the older version of Cucumber too? Versions 7, 6, 5, 4 and even 3, 2, and 1 are still getting thousands of downloads

**Review by davidjgoss — APPROVED** [review]

**Comment by aslakhellesoy** [comment]
Hi @mannyluvstacos,

Thanks for your making your first contribution to Cucumber, and welcome to the Cucumber committers team! You can now push directly to this repo and all other repos under the cucumber organization! 🍾

In return for this generous offer we hope you will:

* ✅ Continue to use branches and pull requests. When someone on the core team approves a pull request (yours or someone else's), you're welcome to merge it yourself.
* 💚 Commit to setting a good example by following and upholding our [code of conduct](https://cucumber.io/conduct) in your interactions with other collaborators and users.
* 💬 Join the [community Slack channel](http://cucumber.io/support#slack) to meet the rest of the team and make yourself at home.
* ℹ️ Don't feel obliged to help, just do what you can if you have the time and the energy.
* 🙋 Ask if you need anything. We're looking for feedback about how to make the project more welcoming, so please tell us!

On behalf of the Cucumber core team,
Aslak Hellesøy
Creator of Cucumber

**Comment by davidjgoss** [comment]
@jan-molak yep I’ll look into that today

**Pull request merged**